src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,115 @@
 import React from 'react';
+import { Todo } from '../../types/Todo';
+import cn from 'classnames';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+enum Status {
+  ALL = 'all',
+  COMPLETED = 'completed',
+  ACTIVE = 'active',
+}
```

Comment:
The `Status` enum is defined here but is also defined in other files like `App.tsx` and `TodoFilter.tsx`. To avoid inconsistencies and duplication, consider importing the `Status` enum from a shared location instead of redefining it.